cmd/skopeo/signing.go

```diff
@@ -64,14 +61,16 @@ func (opts *standaloneSignOptions) run(args []string, stdout io.Writer) error {
 type standaloneVerifyOptions struct {
 }
 
-func standaloneVerifyCmd() cli.Command {
+func standaloneVerifyCmd() *cobra.Command {
 	opts := standaloneVerifyOptions{}
-	return cli.Command{
-		Name:      "standalone-verify",
-		Usage:     "Verify a signature using local files",
-		ArgsUsage: "MANIFEST DOCKER-REFERENCE KEY-FINGERPRINT SIGNATURE",
-		Action:    commandAction(opts.run),
-	}
+	cmd := &cobra.Command{
+		Use:     "standalone-verify MANIFEST DOCKER-REFERENCE KEY-FINGERPRINT SIGNATURE",
+		Short:   "Verify a signature using local files",
+		RunE:    commandAction(opts.run),
+		Example: `skopeo standalone-verify busybox-manifest.json registry.example.com/example/busybox`,
```

Comment:
This example is missing the `KEY-FINGERPRINT`; but on balance I’d probably prefer not giving an example, `standalone-verify` should be actively discouraged for most users.